package.json

```diff
@@ -8,18 +8,24 @@
   "author": "Dat Team",
   "scripts": {
     "start": "electron .",
-    "test": "prettier-standard 'app/**/*.js' 'tests/**/*.js' 'lib/**/*.js' && standard",
-    "watch": "webpack --watch"
+    "test": "prettier-standard 'app/**/*.js' 'tests/**/*.js' 'lib/**/*.js' 'unit-tests/*.js' && npm run unit-tests && standard",
```

Comment:
Can we split lint & unit tests? Also: Order <lint> && <unit> && <lint> is weird.

Reply:
yes i agree, i'll split them and fix it.